index.html

```diff
@@ -0,0 +1,153 @@
+            </div>
+            <nav class="navigation">
+                  <div class="close">X</div>
+                <ul class="nav-items">
+                    <li class="nav-list"><a href="about.html">About</a></li>
+                    <li class="nav-list"><a href="#mainprogramme">Program</a></li>
+                    <li class="nav-list"><a href="#">Join</a></li>
+                    <li class="nav-list"><a href="#">Sponsor </a></li>
+                    <li class="nav-list"><a href="#">News</a></li>
+                </ul>
+            </nav>
+
+        </section>
+
+        <article class="sec2">
+            <div class="mp" id="mainprogramme">
+                <h2 class="mainprogramme">Main Programme</h2>
+            </div>
+            <div class="programmecontainer">
+                <div class="cont">
+                    <img src="images/lecturer.png" alt="lecturer icon"
+                        width="70px" height="auto">
+                    <h3 class="cont-text">lectures</h3>
+                    <div class="cont-description">Come get a talk from the most
+                        professional perspectives in the globe</div>
+                </div>
+                <div class="cont">
+                    <img src="images/exhibition.jpg" alt="lecturer icon"
+                        width="70px" height="auto">
+                    <h3 class="cont-text">Exhibition</h3>
+                    <div class="cont-description">Appreciate various art
+                        excellence on display organized from across
+                        continents</div>
+                </div>
+                <div class="cont">
+                    <img src="images/forum.jpg" alt="lecturer icon" width="70px"
+                        height="auto">
+                    <h3 class="cont-text">Forum</h3>
+                    <div class="cont-description">Come have the time to share
+                        your thoughts or questions with experts on
+                        whose answers you can rely </div>
+                </div>
+                <div class="cont">
+                    <img src="images/workshop.png" alt="lecturer icon"
+                        width="70px" height="auto">
+                    <h3 class="cont-text">Workshop</h3>
+                    <div class="cont-description">There will be a platformwhere
+                        you can build your own work whilst drawing
+                        inspiration from the live scenery</div>
+                </div>
+                <div class="cont">
+                    <img src="images/extra-badge-xxl.png" alt="lecturer icon"
+                        width="70px" height="auto">
+                    <h3 class="cont-text">Extras</h3>
+                    <div class="cont-description">There will be prizes to go
+                        around after the workshop and 3 lucky mentees
+                        taken under each of our guests's wings</div>
+                </div>
+            </div>
```

Comment:
Kindly fix the font-style of the text in this section to match the given design. Also please add the button at the bottom as shown in the design. It would also be better to add the hover action as shown in the design 🤗

Your design:

![image](https://user-images.githubusercontent.com/73714615/190651041-d00ab655-4a68-4cd4-88ca-c815a105b353.png)

Template design:

![image](https://user-images.githubusercontent.com/73714615/190651005-de9fd473-d308-44ed-aef5-25c588a1a4bb.png)

Reply:
done thanks